cmd/manager/main.go

```diff
@@ -57,8 +59,6 @@ import (
 	"github.com/operator-framework/operator-controller/internal/rukpak/preflights/crdupgradesafety"
 	"github.com/operator-framework/operator-controller/internal/rukpak/source"
 	"github.com/operator-framework/operator-controller/internal/version"
-	"github.com/operator-framework/operator-controller/pkg/features"
-	"github.com/operator-framework/operator-controller/pkg/scheme"
```

Comment:
The location of these shouldn't have changed. This will likely fail lint.

Reply:
fixed!